<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WfmPreview</class>
 <widget class="QWidget" name="WfmPreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="B_Close">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>430</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_Plot">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>751</width>
     <height>401</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RB_wfm">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Waveform</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="RB_hist">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>410</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Histogram</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
